pandas/_libs/src/headers/math.h

```diff
@@ -1,11 +0,0 @@
-#ifndef _PANDAS_MATH_H_
```

Comment:
I wouldn't remove this. otherwise windows builds will fail.

Reply:
I see you are trying to fix this using the cpp library below. not sure how this will work out. windows is a bit funky on its incudes.

Reply:
yea I think I'm going to revert it back to math.h even though it's a tad wonky, it was actually working quite well that way.

Reply:
oh ugh I just realized I didn't checkin src/headers/cmath which is why this whole thing is failing.